src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-rate-limit/nfs-rate-limit.component.html

```diff
@@ -0,0 +1,507 @@
+        <ng-template #Export_readbw_Error>
+          <span
+            class="invalid-feedback"
+            *ngIf="rateLimitForm.showError('max_export_read_bw', formDir, 'required')"
+            i18n
+            >This field is required.</span
+          >
+          <span
+            class="invalid-feedback"
+            *ngIf="rateLimitForm.showError('max_export_read_bw', formDir, 'rateByteMaxSize')"
+            i18n
+            >The value is not valid.</span
+          >
+          <span
+            class="invalid-feedback"
+            *ngIf="rateLimitForm.showError('max_export_read_bw', formDir, 'min')"
+            i18n
+            >Enter a positive number.</span
+          >
+        </ng-template>
+      </div>
+      <div class="form-group row">
+        <cds-text-label
+          for="max_export_write_bw"
+          i18n
+          i18n-helperText
+          helperText="Limits the maximum bandwidth that can be used for export write per second"
+          cdRequiredField="Export write bandwidth"
+          [invalid]="
+            rateLimitForm.controls.max_export_write_bw.invalid &&
+            rateLimitForm.controls.max_export_write_bw.dirty
+          "
+          [invalidText]="Export_writebw_Error"
+          >Export Write Bandwidth
+          <input
+            cdsText
+            type="text"
+            placeholder="Export write bandwidth..."
+            i18n-placeholder
+            id="max_export_write_bw"
+            name="max_export_write_bw"
+            formControlName="max_export_write_bw"
+            [invalid]="
+              rateLimitForm.controls.max_export_write_bw.invalid &&
+              rateLimitForm.controls.max_export_write_bw.dirty
+            "
+            defaultUnit="b"
+            defaultEmptyValue="true"
+            cdDimlessBinaryPerSecond
+          />
+        </cds-text-label>
+
+        <ng-template #Export_writebw_Error>
+          <span
+            class="invalid-feedback"
+            *ngIf="rateLimitForm.showError('max_export_write_bw', formDir, 'required')"
+            i18n
+            >This field is required.</span
+          >
+          <span
+            class="invalid-feedback"
+            *ngIf="rateLimitForm.showError('max_export_write_bw', formDir, 'rateByteMaxSize')"
+            i18n
+            >The value is not valid.</span
+          >
+          <span
+            class="invalid-feedback"
+            *ngIf="rateLimitForm.showError('max_export_write_bw', formDir, 'min')"
+            i18n
+            >Enter a positive number.</span
+          >
+        </ng-template>
+      </div>
+    </ng-container>
+
+    <ng-container
+      *ngIf="
+        rateLimitForm?.getValue('enable_qos') &&
+        ((this.type === 'cluster' &&
+          (this.qosTypeVal === 'PerClient' || this.qosTypeVal === 'PerShare_PerClient')) ||
+          (this.type === 'export' &&
+            this.qosTypeVal !== 'PerClient' &&
+            this.qosTypeVal !== 'PerShare'))
```

Comment:
```suggestion
        ((type === 'cluster' &&
          (qosTypeVal === 'PerClient' || qosTypeVal === 'PerShare_PerClient')) ||
          (type === 'export' &&
            qosTypeVal !== 'PerClient' &&
            qosTypeVal !== 'PerShare'))
```

also use consts everywhere